### DISCUSSION TITLE
Remapping cursor movement keys

### DISCUSSION THREAD

**Discussion by johnstegeman** [nvim-neo-tree/neo-tree.nvim#1364] [Q&A] [answered]
I've read https://github.com/nvim-neo-tree/neo-tree.nvim/discussions/1356 - it tells how to map a key to noop, but doesn't answer my question.

I'm using a Colemak keyboard layout, and as such, I use n,u,e,i for cursor movement left, up, down, right. I would like to map the same keys in neo-tree. When I try this:

` window = {
      mappings = { 
              ["e"] = "move_cursor_down",
            },
    },
`

I end up with a warning:

Neo-tree WARN invalid mapping for e  : nil

? is also broken, but strangely, the mapping does work. Is there any way to accomplish this?

**Comment by pysan3** [comment, marked as answer]
I bet you already have `nnoremap e j` in your config.

If you `["e"] = "noop",` in neo-tree config, the global mapping will be applied (or to be precise, neo-tree does nothing) and you should be good to go.

Note that for some keys, they are only assigned source-specific keybinds (like `f` in _filesystem_ to do fuzzy search), so if that's the case, you'll need to "noop" on each source configs (`config.filesystem.window.mappings`) as well.

**Reply by johnstegeman** [reply]
Oh boy - so simple... That was exactly what I needed, thank you!
